tests/integration/test_jd_coverage_gaps.py

```
@@ -0,0 +1,656 @@
+            reasons=["test reason"],
+        )
+        assert isinstance(result.file_path, Path)
+
+
+# ---------------------------------------------------------------------------
+# numbering.py — lines 89, 415-416, 422-423
+# ---------------------------------------------------------------------------
+
+
+class TestNumberingGaps:
+    def _make_scheme_and_generator(self):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_make_scheme_and_generator`

(ANN202)